src/content/docs/apm/agents/go-agent/installation/install-automation-new-relic-go.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: 'Simplify installation with Go instrumentation tool'
+tags:
+  - Agents
+  - Go agent
+  - Installation
+metaDescription: How to use a script to get code suggestions to instrument your Golang apps and services.
+freshnessValidatedDate: 2024-08-20 
+---
+<Callout title="preview">
+  We're still working on this feature, but we'd love for you to try it out!
+
+  This feature is currently provided as part of a preview program pursuant to our [pre-release policies](/docs/licenses/license-information/referenced-policies/new-relic-pre-release-policy).
+</Callout>
+
+Go is a compiled language with an opaque runtime, making it unable to support automatic instrumentation like other languages. For this reason, the New Relic Go agent is designed as an SDK. Since the Go agent is an SDK, it requires more manual work to set up than agents for languages that support automatic instrumentation.
+
+In an effort to reduce that manual effort, the Go agent team created an easy instrumentation tool for Go that is currently in preview. This tool will do most of the work for you by suggesting changes to your source code that instrument your application with the New Relic Go agent.
+
+To get started, check out this [four-minute video](https://asciinema.org/a/r0Il7o2eMiZaLKHIlew3IL2nx), or skip down to [How it works](#how-it-works):
```

Comment:
The colon at the end of this feels odd to me

Reply:
@paperclypse good catch! I replaced that with a period.